fastq_filtrator.py

```diff
@@ -0,0 +1,113 @@
+def read_reads(input_fastq: str) -> list:
+    with open(input_fastq) as fastq_file:
+        reads_fast = fastq_file.readlines()
+
+        all_reads = []
+        seq = 0
+        for read in reads_fast:
+            seq += 1
+            if seq % 2 == 0 and seq % 4 != 0:
+                all_reads.append(read)
+        fastq_file.close()
+
+        return all_reads
+
+
+def gc_read(reads: list, borders=(0, 100)) -> list:
+    reads_count = []
+    for seq in reads:
+        gc_count = 0
+        for nucl in seq:
+            if nucl == 'G' or nucl == 'C':
+                gc_count += 1
+        gc_content = (gc_count / len(seq)) * 100
+        reads_count.append(gc_content)
+
+    filtered_rd = []
+    for count in reads_count:
+        if type(borders) is int:
+            high_bord = borders
+            filtered_rd.append(count < high_bord)

```

Comment:
Границы нужно было включать
```suggestion
            filtered_rd.append(count <= high_bord)

```